pylint/checkers/python3.py

```diff
@@ -83,6 +83,7 @@ def _is_builtin(node):
     "max",
     "min",
     "frozenset",
+    "OrderedDict",
```

Comment:
Mind adding a short test for it? it should live in `unittest_checker_python3`.

Reply:
Added test